scripts/push

```diff
@@ -8,10 +8,11 @@ then
 fi
 
 source "$(dirname $0)/version"
-if [ "$XC_ARCH" == "amd64" ]; then
-  IMAGEID=$( docker images -q openebs/jiva:${VERSION} )
-else 
+if [ ${ARCH} == "linux_arm64" ]; then
```

Comment:
Can we have more generic condition for all other arch except amd64? 
`if [ ${ARCH} != "linux_amd64" ]`

Reply:
I think it is better to fall back to the default (or current) behavior when it is not linux_arm64.